src/shared/components/home/tagline-form.tsx

```diff
@@ -1,14 +1,14 @@
 import { Component, InfernoMouseEvent, linkEvent } from "inferno";
-import { EditSite, GetSiteResponse } from "lemmy-js-client";
+import { EditSite, GetSiteResponse, SiteResponse } from "lemmy-js-client";
 import { i18n } from "../../i18next";
-import { WebSocketService } from "../../services";
-import { capitalizeFirstLetter, myAuth, wsClient } from "../../utils";
+import { RequestState } from "../../services/HttpService";
+import { capitalizeFirstLetter, myAuthRequired, setIsoData } from "../../utils";
 import { HtmlTags } from "../common/html-tags";
 import { Icon, Spinner } from "../common/icon";
 import { MarkdownTextArea } from "../common/markdown-textarea";
 
 interface TaglineFormProps {
-  siteRes: GetSiteResponse;
+  onSaveSite(form: EditSite): Promise<RequestState<SiteResponse>>;
```

Comment:
Same stuff applies here. Don't have this do anything other than pass up the EditSite form.

Reply:
Resolved from changes merged from `use_http_client_dess`.